appserver/admingui/common/src/main/java/org/glassfish/admingui/common/util/GuiUtil.java

```diff
@@ -207,7 +207,24 @@ public static void initSessionAttributes(){
                         .get("extraProperties")).get("asadminRecorderEnabled"));
         sessionMap.put("showDisableAsadminRecorderButton", (boolean)((Map) 
                 ((Map) asadminRecorderEnabled.get("data"))
-                        .get("extraProperties")).get("asadminRecorderEnabled"));        
+                        .get("extraProperties")).get("asadminRecorderEnabled"));  
+
```

Comment:
For setting initial values for the configuration in the admin console. Draws values from the specified defaults.